### ISSUE TITLE
Test WorkspacesNetCore.MetadataReference uses an assembly not available in .NET Core

### ISSUE THREAD

**Issue by sharwell** [dotnet/roslyn#28360]
**Version Used**: 62d5b054ec1086cae7340aad6022f138c97cd06d

Test results (observed using the new test harness):

> Unable to find a type of reference that is appropriate for this file: "WindowsBase".

```text
Test Name:	Roslyn.VisualStudio.IntegrationTests.Workspace.WorkspacesNetCore.MetadataReferenceAsync (VS2017)
Test FullName:	Roslyn.VisualStudio.IntegrationTests.Workspace.WorkspacesNetCore.MetadataReferenceAsync
Test Source:	D:\dev\roslyn\src\VisualStudio\IntegrationTest\IntegrationTests\Workspace\WorkspacesNetCore.cs : line 30
Test Outcome:	Failed
Test Duration:	0:00:05.246

Result StackTrace:	
at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.Package.Automation.OAVSReferences.Add(String bstrPath)
   at Microsoft.VisualStudio.IntegrationTest.Utilities.InProcess2.SolutionExplorer_InProc2.AddMetadataReference(String assemblyName, String projectName) in D:\dev\roslyn\src\VisualStudio\IntegrationTest\TestUtilities\InProcess2\SolutionExplorer_InProc2.cs:line 77
   at Roslyn.VisualStudio.IntegrationTests.Workspace.WorkspaceBase.<MetadataReferenceAsync>d__9.MoveNext() in D:\dev\roslyn\src\VisualStudio\IntegrationTest\IntegrationTests\Workspace\WorkspaceBase.cs:line 55
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Roslyn.VisualStudio.IntegrationTests.Workspace.WorkspacesNetCore.<MetadataReferenceAsync>d__2.MoveNext() in D:\dev\roslyn\src\VisualStudio\IntegrationTest\IntegrationTests\Workspace\WorkspacesNetCore.cs:line 41
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Microsoft.VisualStudio.IntegrationTest.Utilities.Harness.IdeTestInvoker.<ScreenshotOnFailureAsync>d__2.MoveNext() in D:\dev\roslyn\src\VisualStudio\IntegrationTest\TestUtilities\Harness\IdeTestInvoker.cs:line 43
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
Result Message:	System.ArgumentException : Unable to find a type of reference that is appropriate for this file: "WindowsBase".
```

**Comment by sharwell** [comment]
Reassigning to @ivanbasov since #28173 was rejected.

**Comment by sharwell** [comment]
Fixed by 2f71dac1
